grim.go

```diff
@@ -146,6 +151,17 @@ func (i *Instance) BuildNextInGrimQueue(logger *log.Logger) error {
 	return nil
 }
 
+func repoIsAlive(hook *hookEvent) bool {
+	url := hook.URL
+	if len(url) >= 12 {
+		lastCommitNum := url[len(url)-12:]
+		if lastCommitNum == "000000000000" { //if last 12 digits is zero, it's a deleted branch
```

Comment:
It seems very suspicious to me that this is the best way we can detect deleted events.  I want @bhand-mm to code review and verify.